### DISCUSSION TITLE
Remark42 out of the box in Ghost

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#1117] [Ideas]
Ghost is a good blogging CMS, it would be great if Remark42 made a plugin for this CMS.

https://ghost.org/integrations/?tag=community

Most likely, nothing will prevent me from adding Remark42 myself, but for many it will be a problem

**Comment by seabits-steve** [comment]
Ghost doesn't use plugins. You can easily add Remark42 to Ghost by using the code examples they provide and adding it to your theme. I've done it myself and it works great.
